res/values/cr_strings.xml

```diff
@@ -1020,4 +1020,7 @@
     <!-- Battery temperature -->
     <string name="battery_temperature">Battery temperature</string>
 
+    <!-- Qs Transparency -->
+    <string name="qs_transparency_title">Qs Transparency</string>
+    <string name="qs_transparency_summary">Choose how transparent you want your qs to be (Requires SystemUI Restart)</string>
```

Comment:
QS also
and SystemUI restart is not quite easy to do as we don't have dedicated button
Maybe say System restart ?

Reply:
Ok